unstoppable-ios-app/domains-manager-ios/Modules/DomainsCollection/Cells/DomainsCollectionCarouselCardCell/DomainsCollectionCarouselCardCell.swift

```diff
@@ -220,6 +220,9 @@ private extension DomainsCollectionCarouselCardCell {
             case .normal:
                 setIndicatorStyle(nil)
                 setStatusMessageComponent(nil)
+            case .newNonInteractable:
+                setIndicatorStyle(nil)
+                setStatusMessageComponent(nil)
```

Comment:
Would be simpler to combine .normal and .newNonInteractable cases in line coma separated

Reply:
I used to have this in code, but then it not possible to debug the new case. Leaving it separated